doc/PAGES/ProgrammersManual/CodingTutorials/CodeTut1.h

```diff
@@ -49,7 +49,7 @@ MeshCreateUnpartitioned2DOrthoMesh(mesh,mesh)
 VolumeMesherExecute();
 
 --############################################### Set Material IDs
-VolumeMesherSetMatIDToAll(0)
+mesh.SetMatIDToAll(0)
```

Comment:
didn't Zach have another name for this one? with uniform or something like that?

Reply:
I did not make that change, yet.